internal/cmd/launcher.go

```diff
@@ -225,6 +233,21 @@ func k6buildProvision(gs *state.GlobalState, deps k6deps.Dependencies) (commandE
 	return &customBinary{binary.Path}, nil
 }
 
+func getBinaryCacheDir(fs fsext.Fs) (string, error) {
+	//nolint:forbidigo
+	// TODO: init this as part of the GlobalState
+	// and remote it from here
+	userdir, err := os.UserCacheDir()
+	if err != nil {
```

Comment:
I'm not sure we should return an error here. In the GlobalState when [we fail to get the config dir](https://github.com/grafana/k6/pull/4743), we default to `.config`. I prefer to be consistent in how we handle those errors.